src/components/Article/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <article className="Article">
+    <h1 className="Article__title">
+      {title}
+    </h1>
+    <span className="Article__date">
+      {date}
+    </span>
+    <p className="Article__text">
+      {text}
+    </p>
+  </article>
```

Comment:
You don't style any of the elements here, so adding classNames is not necessary.